<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCollectionConfirmDialog</class>
 <widget class="QDialog" name="CCollectionConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_CollectAffirm_collectAddr">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>归币地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Amount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0 BTC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cancel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_AddressCount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_CollectAffirm_num">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>总数量：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Addr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_CollectAffirm_googleCode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>谷歌验证码：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_CollectAffirm_addrNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>待生成地址数量：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>10</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>确认生成</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_TxFee">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.005</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Verify_code">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_CollectAffirm_MinersFee">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>总矿工费：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OK">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
